guides/database/data-manipulation/README.md

```diff
@@ -1,49 +1,76 @@
+Database objects are created with the native program code operator `new`. For example, consider the following database class:
 
-<div class="code-name code-title">Create a database class</div>
 ```cs
 [Database]
-public class Employee
+public class Person
 {
     public String FirstName;
     public String LastName;
-    public Department Department;
-    public Employee Manager;
 }
 ```
 
-<div class="code-name code-title">Create, update, delete object</div>
+To create a new instance of this class, the syntax `new Person()` would be used, like this:
+
+```cs
+var person = new Person()
+{
+    FirstName = "John",
+    LastName = "Doe"
+};
+```
+
+## Update Database Objects
+
+A database object can be updated using the native program code assign operator `=`.
+
+For example, instead of instantiating an object like in the example above, it's possible to create the object and then update its properties and fields:
+
+```cs
+var person = new Person();
+person.FirstName = "John";
+person.LastName = "Doe";
+```
+
+To update the `LastName` of all the `Person` objects in the database, they would be looped through and updated, like so:
+
 ```cs
-Db.Transact(() =>
+var people = Db.SQL<Person>("SELECT p FROM Person p);
+foreach (var person in people)
 {
-    // Creating database object.
-    Employee emp = new Employee();
-    // Updating database object.
-    emp.FirstName = "John";
-    emp.LastName = "Smith";
-    // Deleting database object.
-    emp.Delete();
-});
+    person.LastName = person.LastName.ToUpper();
+}
 ```
-The following example shows how to update the `LastName` of all
-`Employee` objects to upper case.
 
-<div class="code-name code-title">Update properties in loop</div>
+## Delete Database Objects
+
+There are two ways to delete database objects in Starcounter:
+
+1. Using the `Delete` method on a specific object
+2. Using `DELETE FROM` in `Db.SlowSQL`
+
+Essentially, `Delete` is used for single objects and `DELETE FROM` is used for many objects. 
+
+In code, they look like this:
+
 ```cs
-Db.Transact(() =>
-{ 
-    string query = "SELECT e FROM Employee e";
-    foreach (Employee emp in Db.SQL<Employee>(query))
-    {
-        emp.LastName = emp.LastName.ToUpper();
-    }
-});
+person = new Person();
+person.Delete();
+
+Db.SlowSQL("DELETE FROM Person");
 ```
+
+Here, `person.Delete()` will just delete that single object while `DELETE FROM Person` will delete all objects belonging to the `Person` class. 
+
+It's worth keeping in mind that using `DELETE FROM` might create massive transactions that can hit the size limit for transactions as explained in the [Kernel Q&A](guides/working-with-starcounter/kernel-q-and-a/). To work around this limit, create a loop that deletes each single object with the `Delete` method and commits the transaction before it gets too large. The performance loss for this is small since `DELETE FROM` is implemented in a similar fashion except that it does not break up the transaction into smaller transactions.
```

Comment:
In the case of your examples you provide wrong statement. Identifier `Person` is unknown in the where clause. It will be the same problem in select statement.